### ISSUE TITLE
Why am I getting a malware warning on Windows 11 from Microsoft Defender when visiting the website?

### ISSUE THREAD

**Issue by fguendling** [chriskuehl/fluffy#175]
Would be great to get some information. Thanks

**Comment by fguendling** [comment]
![image](https://github.com/chriskuehl/fluffy/assets/46090606/2a4f6bf2-4e9d-4083-9ab5-517c0e1ded18)

**Comment by chriskuehl** [comment]
There are some details in #79. Basically, the upload policy used to be very open (it was only used by me and my friends) and someone started distributing malware through it. As a result the domain was put on various security vendors's blocklists.

I spent a bunch of time adding upload restrictions, virus scanning, and other features to try to prevent malware which has more or less worked, but it seems impossible to get off these lists permanently. I've submitted a bunch of removal requests to various companies to request the domain be unblocked. Most of the requests are successful, but then a few months later it's blocked again for reasons unknown to me. I gave up and usually don't bother anymore now that the main usage of fluffy is internally at companies.

As far as I know there is no malware hosted now but there's not much I can do about the blocking.
